independent-publisher-connectors/eBay/apiDefinition.swagger.json

```diff
@@ -0,0 +1,6241 @@
+                        "value": {
+                          "type": "string",
+                          "description": "A string representation of a dollar value expressed in the currency field.",
+                          "title": "Value"
+                        }
+                      },
+                      "description": "This field is needed if the Strikethrough Pricing feature will be used in the offer.",
+                      "title": "OriginalRetailPrice"
+                    },
+                    "price": {
+                      "type": "object",
+                      "properties": {
+                        "currency": {
+                          "type": "string",
+                          "description": "A three-digit string value representing the currency type.",
+                          "title": "Currency"
+                        },
+                        "value": {
+                          "type": "string",
+                          "description": "A string representation of a dollar value expressed in the currency field.",
+                          "title": "Value"
+                        }
+                      },
+                      "description": "Listing price of the product.",
+                      "title": "Price"
+                    },
+                    "pricingVisibility": {
+                      "type": "string",
+                      "description": "This field is needed if Minimum Advertised Price feature will be used in the offer.",
+                      "title": "Pricing Visibility"
+                    }
+                  },
+                  "description": "This container shows the listing price for the product offer, and if applicable, the settings for the Minimum Advertised Price and Strikethrough Pricing features."
+                },
+                "quantityLimitPerBuyer": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "This field is only applicable if the seller wishes to set a restriction on the purchase quantity of an inventory item per seller.",
+                  "title": "Quantity Limit Per Buyer"
+                },
+                "secondaryCategoryId": {
+                  "type": "string",
+                  "description": "Unique identifier for a secondary category and applicable if seller decides to list the item under two categories.",
+                  "title": "Secondary Category Id"
+                },
+                "sku": {
+                  "type": "string",
+                  "description": "Seller-defined SKU value of the product in the offer.",
+                  "title": "SKU"
+                },
+                "status": {
+                  "type": "string",
+                  "description": "The value specifies the status of the offer.",
+                  "title": "Status"
+                },
+                "storeCategoryNames": {
+                  "type": "array",
+       
```

Comment:
@artesianst can you please explain this empty path. is it possible to remove it please?